virtio-devices/src/block.rs

```diff
@@ -521,6 +581,15 @@ impl VirtioDevice for Block {
                     ActivateError::BadActivate
                 })?;
 
+            let rate_limiter: Option<RateLimiter> = self
+                .rate_limiter_config
+                .map(RateLimiterConfig::try_into)
+                .transpose()
+                .map_err(|e| {
+                    error!("failed to create RateLimiter from RateLimiterConfig: {}", e);
+                    ActivateError::CreateRateLimiter
```

Comment:
Does it make sense to expose that error though `ActivateError::CreateRateLimiter(e)`

Reply:
Yes, I think that's a better idea. I was too focused on following the same style of error handling nearby. Fixed.